samples/python/whisper_speech_recognition/whisper_speech_recognition.py

```diff
@@ -6,12 +6,10 @@
 import openvino_genai
 import librosa
 
-
 def read_wav(filepath):
     raw_speech, samplerate = librosa.load(filepath, sr=16000)
     return raw_speech.tolist()
 
-
```

Comment:
Having two lines between functions is a common code style. See PEP8. Please, revert it.

Reply:
Got it.